spec/rails_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
+require "support/auth_helper"
```

Comment:
When you uncomment line 26 `# Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }` it would automatically require all files from `support` directory